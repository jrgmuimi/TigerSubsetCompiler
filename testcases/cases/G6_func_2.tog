let
    function add(a:int, b:int):int = 
        return a+b
    end
    function sub(a:int, b:int):int = 
        return a-b
    end
    function mul(a:int, b:int):int = 
        return a*b
    end
    function div(a:int, b:int):int = 
        return a/b
    end
in
    printint( add( 2, 3) );
    printstring("\n");
    printint( sub( 2, 6) );
    printstring("\n");
    printint( div( 4, 2) );
    printstring("\n");
    printint( mul( 2, 3) );
    printstring("\n")
end