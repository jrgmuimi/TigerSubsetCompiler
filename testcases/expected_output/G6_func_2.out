MARS 4.5  Copyright 2003-2014 Pete Sanderson and Kenneth Vollmar

5
-4
2
6

